perllib/FixMyStreet/Cobrand/UKCouncils.pm

```diff
@@ -501,6 +501,14 @@ sub updates_disallowed {
     return $self->next::method(@_);
 }
 
+# Report if cobrand denies updates by user
+sub deny_updates_by_user {
+    my $self = shift;
+    my $cfg = $self->feature('updates_allowed') || '';
+    if ($cfg eq 'none' || $cfg eq 'staff') {
+        return 1;
```

Comment:
I should have mentioned this but we probably need to handle when updates_allowed is set to 'open' which means they can leave an update if the report is open but not if it's fixed or closed.

There's also a reporter-open and reporter settings, the first of which is probably relevant.